package.json

```diff
@@ -42,7 +42,6 @@
     "get-next-version": "node scripts/get-next-version.js",
     "postinstall": "node ./scripts/run-postInstall.js",
     "lint": "lerna run lint --no-bail --concurrency 2",
-    "lint-changed": "lint-changed",
```

Comment:
Did you log an issue to fix the broken script? Or remove in a MVB? Find with removing but if it's not working and its from something we ship, seems like this will get lost.

Reply:
I assumed that because the tests in `eslint-plugin-dev` pass, that the scripts aren't totally broken, but on second glance I actually cannot successfully invoke them no matter what I try. It looks like @cypress-io/tools is looking at removing `eslint-plugin-dev` entirely, so I commented to ask if we can blast these scripts into orbit as part of that change: https://github.com/cypress-io/cypress/issues/24251#issuecomment-1367487970